packages/ava/src/__tests__/data/utils/isType/isDateString.test.ts

```diff
@@ -0,0 +1,54 @@
+import { getIsoDatePatterns, getIsoTimePatterns, isDateString } from '../../../../data/utils/isType/isDateString';
+
+describe('func: isDateString', () => {
+  test('check a string is a date', () => {
+    expect(isDateString('1991')).toBe(true);
+    expect(isDateString('1890')).toBe(true);
+    expect(isDateString('1722')).toBe(false);
+    expect(isDateString('3013')).toBe(false);
+    expect(isDateString('2010 W08 1')).toBe(true);
```

Comment:
we need docs to show users the logic of this func or the apis dep on this.

could have a @todo